### PR TITLE
Add documentation for OSE 3.1 to 3.0 downgrade.

### PR DESCRIPTION
**Pull request by dgoodwin**
Documents the process of reverting to 3.0 rpms after an upgrade, and restoring
etcd, and various configuration files from backups.

Apologies as this is still a little rough around the edges but I wanted to get review started ASAP so I can get this into line quickly.

Ping @adellape

### CODE REVIEW

**Comment by dgoodwin** [comment]
Prettier version: http://file.rdu.redhat.com/~dgoodwin/docs/downgrade/admin_guide/downgrade.html

File: admin_guide/downgrade.adoc [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+====
+----
+$ mv $ETCD_DIR /var/lib/etcd.orig
+$ cp -Rp /var/lib/openshift/etcd-backup-20151120093517/ $ETCD_DIR
+$ chcon -R --reference /var/lib/etcd.orig/ $ETCD_DIR
+$ chown -R etcd:etcd $ETCD_DIR
+----
+====
+
+If you are using a separate etcd cluster, you should also restore /etcd/etcd/etcd.conf from backup or .rpmsave.
```
Comment (adellape) [inline admin_guide/downgrade.adoc]: Drive-by comment (will review doc more fully ASAP):

Think this should be /etc/etcd/etcd.conf (has an extra "d" currently).

Comment (dgoodwin) [inline admin_guide/downgrade.adoc]: Fixed along with another typo QE caught.

**Comment by dgoodwin** [comment]
Bump for review whenever you get a chance @adellape

File: _build_cfg.yml [hunk 2]
```diff
@@ -243,6 +243,8 @@ Topics:
     File: building_dependency_trees
   - Name: Troubleshooting Networking
     File: sdn_troubleshooting
+  - Name: Downgrading to 3.0
+    File: downgrade
```
Comment (adellape) [inline _build_cfg.yml]: Since this is specific to OSE, we'll need to add a distro after the `File` here so that the topic only shows up in the OSE docs and not Origin.

```
  - Name: Downgrading to 3.0
    File: downgrade
    Distros: openshift-enterprise
```

**Comment by adellape** [comment]
Read through and didn't have any issues that wouldn't be taken care of during normal follow-up edits. I still want to walk through and test the steps myself, but I see QE didn't have issues with them, so I'm good with merging once the one in-line comment is handled, and I can follow up with format/style edits while testing it out.

**Comment by dgoodwin** [comment]
Distros now specified. Thanks!

**Comment by dgoodwin** [comment]
@adellape Removed WIP, QE has had a look, should be good to go on my end.

**Comment by dgoodwin** [comment]
Added sed commands to patch in the --force-new-cluster option for etcd startup, and then remove it, when I saw this in QE's test steps.

**Comment by adellape** [comment]
:+1:
